credential/base/src/main/java/org/wildfly/security/password/interfaces/DigestPassword.java

```diff
@@ -90,10 +90,10 @@ public interface DigestPassword extends OneWayPassword {
      */
     byte[] getDigest();
 
+    @Override
     default DigestPasswordAlgorithmSpec getParameterSpec() {
         return new DigestPasswordAlgorithmSpec(getUsername(), getRealm());
     }
-
```

Comment:
@kanha-gupta Just a minor, the new line should be kept here since it separates the 2 methods and makes it more readable